### PR TITLE
[58262] Introduce CI for the Python starter

### PR DESCRIPTION
**Pull request by akashchi**

### CODE REVIEW

File: automation/snyk/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,38 @@
+FROM ${OPENVINO_UBUNTU_IMAGE}
+
+ARG http_proxy
```
Comment (artyomtugaryov) [inline automation/snyk/Dockerfile:3]: I believe, we can remove proxy args

File: automation/snyk/docker-entrypoint.sh [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+runCmdAsDockerUser() {
+  su ${USER_NAME} -m -c "$1"
+
+  return $?
+}
+
+runCmdAsDockerUser "cat ${MONITOR_OUTPUT_FILE} | jq -r 'if type==\"array\" then .[].uri? else .uri? end' | awk '{print \"<center><a target=\\\"_blank\\\" href=\\\"\" \$0 \"\\\">View On Snyk.io</a></center>\"}' > \"${HTML_FILE}\" 2>>\"${ERROR_FILE}\""
+
+runCmdAsDockerUser "cat \"${OUTPUT_FILE}\" | \
+jq 'def sortBySeverity: .vulnerabilities|= map(. + {severity_numeric: (if(.severity) == \"high\" then 1 else (if(.severity) == \"medium\" then 2 else (if(.severity) == \"low\" then 3 else 4 end) end) end)}) |.vulnerabilities |= sort_by(.severity_numeric) | del(.vulnerabilities[].severity_numeric); if (. | type) == \"array\" then map(sortBySeverity) else sortBySeverity end'| \
```
Comment (artyomtugaryov) [inline automation/snyk/docker-entrypoint.sh:47]: whaat?

File: automation/snyk/Dockerfile (hunk 1 above)
Comment (akashchi) [reply automation/snyk/Dockerfile:3]: I think it is needed later in the ENV for the Snyk.

File: automation/Jenkinsfile [hunk 3]
```diff
@@ -0,0 +1,3 @@
+@Library('workbench_lib') _
```
Comment (artyomtugaryov) [inline automation/Jenkinsfile:1]: You haven't checked it.

File: automation/snyk/docker-entrypoint.sh (hunk 2 above)
Comment (akashchi) [reply automation/snyk/docker-entrypoint.sh:47]: A magic thing introduced by @SalnikovIgor 
It is working in the wb CI, though.

File: starting_package/requirements-linux.txt [hunk 4]
```diff
@@ -1 +1 @@
-psutil==5.7.0
+psutil==5.8.0
```
Comment (akashchi) [inline starting_package/requirements-linux.txt:1]: Reason: https://github.com/giampaolo/psutil/issues/1753

File: automation/snyk/docker-entrypoint.sh (hunk 2 above)
Comment (SalnikovIgor) [reply automation/snyk/docker-entrypoint.sh:47]: I think there is no need to build a docker image for these python requirements. Just use snyk:python image

File: automation/scripts/parse_snyk_output.py [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+    args = parser.parse_args()
+
+    return args
+
+
+def parse_output(path_to_raw_output: str, path_to_result_file: str) -> int:
+    with open(path_to_raw_output, 'r', encoding='utf-8') as output_file:
+        data = output_file.read()
+        json_start = data.find('{')
```
[the remark this answers is not in the document]
Comment (akashchi) [reply automation/scripts/parse_snyk_output.py]: Refactored in the repo with tests.

File: automation/scripts/run_snyk_scan.sh [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+      ;;
+    *)
+      echo Unsupport argument $1
+      exit 1
+      ;;
+  esac
+done
+
+OUTPUT_FILE_RAW=snyk-result.txt
+pushd ${PROJECT_PATH}
```
[the remark this answers is not in the document]
Comment (akashchi) [reply automation/scripts/run_snyk_scan.sh]: Refactored in the repo with tests.

**Review by demid5111 — APPROVED** [review]

**Review by artyomtugaryov — APPROVED** [review]

**Pull request merged**